en/orm/validation.rst

```diff
@@ -437,6 +437,13 @@ two rules::
     // Between 3 and 5 tags
     $rules->add($rules->validCount('tags', 3, '>=', 'You must have at least 3 tags'));
     $rules->add($rules->validCount('tags', 5, '<=', 'You must have at most 5 tags'));
+    $rules->add($rules->validCount('subscription', 0, '==', 'You may not have a subscription'));
+
+.. note::
+
+   ``validCount`` returns ``false`` if the property is not countable or does not exist.
+   E.g. comparing via ``<``, ``<=`` or against ``0`` will return ``false``,
+   if you do not supply at least an empty list of - say - subscriptions.
```

Comment:
I still think that the first sentence is sufficient. 

The second sentence is unclear to me, it sounds like it could return true if you used a different comparison.